module/Api/src/Domain/QueryHandler/Messaging/Message/ByConversation.php

```diff
@@ -39,13 +39,16 @@ public function handleQuery(QueryInterface $query)
 
         $messages = $messageRepository->fetchPaginatedList($messagesQuery);
 
+        /** @var MessagingConversation $conversation */
         $conversation = $this->getRepo(Repository\Conversation::class)->fetchById($query->getConversation());
 
         $this->markMessagesAsReadByCurrentUser($messages);
 
         return [
             'result'       => $messages,
             'count'        => $messageRepository->fetchPaginatedCount($messagesQuery),
+            'licence'      => $conversation->getRelatedLicence()->serialize(),
+            'application'  => $conversation->getTask()->getApplication()->serialize(),
```

Comment:
Sometimes we have licences without applications, so maybe add some fallback here to prevent calls to serialize() on null